shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -16,7 +16,11 @@
       {{/if}}
     {{/if}}
   {{else}}
-    <a>Sign in ▾</a>
+    {{#if isCurrentRoute 'shared'}}
+      <a>Don't lose this link! Sign up for free ▾</a>
```

Comment:
On personal servers, fwiw, it's not super-duper clear to a random visitor that it's a "personal server". Imagine me giving a link to rose.sandcats.io to a friend who has no idea what Sandstorm is, but now knows that rose.sandcats.io is some kind of thing that has collaborative text documents. Might as well encourage them to sign in there, too.

So IMHO emphasizing that signing up is "free" is likely a good idea even there.